### DISCUSSION TITLE
I think it's an error or a bug

### DISCUSSION THREAD

**Discussion by Ajxrhaider** [aniyomiorg/aniyomi#956] [Q&A]
Since the last update, almost all the extensions for manga stopped being recognized even if they are installed
What's worse is that the app can no longer download manga extensions as it can't load extensions list
Is there anything that can be done

**Comment by Alex178190282** [comment]
Well brother I advise you to make a copy of your aniyomi manga library and download tachiyomi and go from there reading manga and when they finally update Aniyomi and fix the problem go back to reading manga there or you can just stop reading manga for a while.

**Comment by intvrtdntcse** [comment]
This was suggested by another person, it worked for me:
1 - Create a backup from Aniyomi (or Tachiyomi)
2 - Download and install this version of Aniyomi: : Preview r6141 (0.14.3-6141)
3 - Restore the backup using the file you created in step # 1

I hope this helps.
